src/main/java/net/sf/jabref/model/util/RemoveLatexCommands.java

```diff
@@ -1,12 +1,7 @@
-package net.sf.jabref.logic.layout.format;
+package net.sf.jabref.model.util;
 
-import net.sf.jabref.logic.layout.LayoutFormatter;
-import net.sf.jabref.logic.util.strings.StringUtil;
+public class RemoveLatexCommands {
 
-public class RemoveLatexCommands implements LayoutFormatter {
```

Comment:
As I in that case had to move the interface to model as well, which didn't really make sense just to have that single function in model. The actual layout formatter is still there in logic though, calling the class in model.